### PR TITLE
Optimizes shadow compute caster scene bound

### PR DESCRIPTION
**Pull request by Kuranes**
Do it once per shadowed scene instead of every shadowmap

### CODE REVIEW

**Comment by Kuranes** [comment]
Perf improvemvent of doing computeBounds once instead of N times per shadow counts for a (N-1)*50% of shadow js pass improvement.
For example https://sketchfab.com/models/9df1dc92bcbd4ca2a911c878e212e88c/edit?debug3d=1

the CullShadowCasting block code goes from:
desktop ~3ms => ~0.88 
mobile   ~10ms => ~3.5ms

File: sources/osgShadow/ShadowMap.js [hunk 1]
```diff
@@ -930,14 +873,6 @@ ShadowMap.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( S
             //console.log( 'shadow early OUT' );
```
Comment (cedricpinson) [inline sources/osgShadow/ShadowMap.js]: you can remove the console.log comment

Comment (cedricpinson) [inline sources/osgShadow/ShadowMap.js]: the comment in aimShadowCastingCamera https://github.com/cedricpinson/osgjs/pull/790/files?diff=split#diff-8cf8f17e8132314edcfa57e87eda7ab4R686 says that it's a frustrum, but the variable is bbox, is it accurate or should be a renamed of variable and comments ?

File: sources/osgShadow/ShadowedScene.js [hunk 2]
```diff
@@ -99,24 +178,70 @@ ShadowedScene.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInheri
...
+                if ( !st || !st.valid() ) continue;
+
+                if ( st.isDirty() ) {
+                    isDirty = true;
+                    st.init();
+                }
+
+                if ( st.isEnabled() || !st.isFilledOnce() ) {
+                    isDirty = true;
+
```
Comment (cedricpinson) [inline sources/osgShadow/ShadowedScene.js]: remove the blank line

File: sources/osgShadow/ShadowMap.js (hunk 1 above)
Comment (Kuranes) [inline sources/osgShadow/ShadowMap.js]: Will rename as bbox 

( The bbox is used in as the frustumBounds, doesn't inmply it's a frustum, but that was when I was doing ops on it in that function, which not the case anymore)

**Pull request merged**